docker-compose.yaml

```
@@ -0,0 +1,49 @@
+services:
+  api:
+    image: pramsey/pg_featureserv:20240305
+    ports:
+      - "0.0.0.0:${OSM_PG_FEATURESERV_API_SYSTEM_PORT}:9000"
+    environment:
+      - DATABASE_URL=postgres://${OSM_PG_FEATURESERV_POSTGRES_USER}:${OSM_PG_FEATURESERV_POSTGRES_PASS}@db/${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
```

```suggestion
      - DATABASE_URL=postgres://${OSM_PG_FEATURESERV_POSTGRES_USER}:${OSM_PG_FEATURESERV_POSTGRES_PASS}@db/${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
      - PGFS_WEBSITE_BASEMAPURL=https://tile.openstreetmap.de/{z}/{x}/{y}.png

```

add a basemap to pg_featureserv